.errcheck-excluded.txt

```diff
@@ -0,0 +1,2 @@
+os.MkdirAll
```

Comment:
since it's just 2 lines or possibly even 1, let's just drop this file if possible and call errcheck with --flags. it's kinda annoying to have an extra file in repo root just for this.

Reply:
yes, dropped. Will push soon.

Reply:
pushed.